Login.js

```diff
@@ -0,0 +1,89 @@
+import React, {Component} from 'react';
+import {
+  StyleSheet,
+  Text,
+  View,
+  StatusBar,
+  Image,
+  TextInput,
+  TouchableOpacity,
+  Button,
+} from 'react-native';
+
+export default class Login extends Component {
+  render() {
+    return (
+      <View style={styles.container}>
+        <View style={styles.logoContainer}>
+          <Image
+            style={{width: 350, height: 85}}
+            source={require('./public/logo.png')}
+          />
+          <Text> Welcome to ClimbAR </Text>
+        </View>
+        <TextInput
+          style={styles.inputBox}
+          placeholder="Email"
+          placeholderTextColor="#e4572e"
+        />
+        <TextInput
+          style={styles.inputBox}
+          placeholder="Password"
+          secureTextEntry={true}
+          placeholderTextColor="#e4572e"
+        />
+        <TouchableOpacity style={styles.button}>
+          <Text style={styles.buttonText}> Login </Text>
```

Comment:
The button doesn't do anything yet right?

Reply:
the button should take you to the home screen after clicking it

Reply:
@nries1 oh nevermind I see that the new commits didn't get pushed through

Reply:
oh nevermind it did get pushed through, think you just might looking at an old branch